polkadot/xcm/xcm-builder/src/location_conversion.rs

```diff
@@ -311,6 +311,8 @@ impl<Network: Get<Option<NetworkId>>, AccountId: From<[u8; 32]> + Into<[u8; 32]>
 			MultiLocation { parents: 0, interior: X1(AccountId32 { id, network }) }
 				if network == Network::get() =>
 				id,
+			MultiLocation { parents: 0, interior: X1(Plurality { id: BodyId::Treasury, .. }) } =>
+				PalletId(*b"py/trsry").into_account_truncating(),
```

Comment:
You should probably add this on a new struct implementing `ConvertLocation`, since this one is for `AccountId32`s

Reply:
^^ That, except remove "probably".

Reply:
got it.